com.maplerain.telsis_translator_flutter.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
```

Comment:
You should maintain this file in your repo and distribute it in the tarball.

Reply:
Yes, it is in the tarball. I saw this file on other repos and blindly followed. My apologies. 🙇‍♀️